docs/source/development/getting-started.rst

```diff
@@ -0,0 +1,130 @@
+===============
+Getting Started
+===============
+
+We’re pleased that you are interested in working on Junction.
+
+This document is meant to get you setup to work on Junction and to act as a
+guide and reference to the the development setup. If you face any issues during
+this process, please `open an issue`_ about it on the issue tracker.
+
+
+Initial Setup
+=============
+
+Junction's development workflow is automated using `nox`_. Thus, you need
+the ``nox`` command to be installed on your system. We recommend using ``pipx``
+to install ``nox`` in its own isolated environment.
+
+.. code-block:: console
+
+   $ python -m pip install pipx
+   $ pipx install nox
+
+You will need to have a working Redis server on your system. You may
+additionally need PostgreSQL and TCL as well.
```

Comment:
Additionally, as in -- is this optional?

If no, I think it can be better reworded as - "Other than this, you would also need a working Redis server on your system alongwith PostreSQL, and TCL installed."

If yes, I guess, we should mention when would someone want to install them.

Reply:
Postgresql is used n production server. For dev sqlite is the default. But anyone who wants to use psql can do so. Tcl was earlier a dependency for psql, but now it comes packaged with it I think